ref/vk/common_geometry.c

```diff
@@ -199,3 +199,145 @@ void GL_SubdivideSurface( model_t *loadmodel, msurface_t *fa )
 	// do subdivide
 	SubdividePolygon_r( loadmodel, fa, fa->numedges, verts[0] );
 }
+
+/*
+================
+VK_LightmapCoord
+
+Total copypaste of R_LightmapCoord from gl_rsurf.c
+================
+*/
+void VK_LightmapCoord( const vec3_t v, const msurface_t *surf, const float sample_size, vec2_t coords )
```

Comment:
Поскольку это само по себе не вулкано-специфичное:
```suggestion
R_LightmapCoord

Total copypaste of R_LightmapCoord from gl_rsurf.c
================
*/
void R_LightmapCoord( const vec3_t v, const msurface_t *surf, const float sample_size, vec2_t coords )
```